components/org.wso2.carbon.identity.handler.event.account.lock/src/main/java/org/wso2/carbon/identity/handler/event/account/lock/AccountLockHandler.java

```diff
@@ -337,7 +337,8 @@ protected boolean handlePostAuthentication(Event event, String userName, UserSto
             if (AccountUtil.isAccountLockByPassForUser(userStoreManager, userName)) {
                 if (log.isDebugEnabled()) {
                     String bypassMsg = String.format("Account locking is bypassed as lock bypass role: %s is " +
-                            "assigned to the user %s", AccountConstants.ACCOUNT_LOCK_BYPASS_ROLE, userName);
+                                    "assigned to the user %s", AccountConstants.ACCOUNT_LOCK_BYPASS_ROLE,
```

Comment:
Remove the indentation. Check other places.

Reply:
This indentation was introduced by the WSO2 code formatter [1]. Do we need to remove it?

[1] - https://github.com/wso2/product-is/wiki/Contributing-to-Identity-Server#how-to-configure-wso2-code-style-template

Reply:
This code style refers to something like below.
```
obj.method(ar+
        g1,
    arg2,
    arg3,
    arg4)
```